dist.sh

```diff
@@ -70,16 +78,27 @@ for platform in $platforms; do
 
     cp -r $template_dir/* $dist_path/
 
-    cp $core_dir/build/libs/*.jar $dist_path/workcraft.jar
+    # resource_dir is simply the distribution root. OS X needs special care.
+    resource_dir=$dist_path
```

Comment:
I have removed the `resource_dir` in the coming update and restored the use of `dist_path`. `dist_path` is now updated to the Resources directory after the `sed` command.